### PR TITLE
Show allowed file extensions in FileManager

### LINKED ISSUE #4218: File Manager: Display file extensions
Displaying allowed file extensions in FileManager is currently missing:

image

### PR DESCRIPTION
**Pull request by panagiotappl**
closes #4218 

image

### CODE REVIEW

File: indico/modules/events/editing/client/js/components/FileManager/index.jsx [hunk 1]
```diff
@@ -91,6 +93,13 @@ function FileType({uploadURL, fileType, uploads}) {
   return (
     <div styleName="file-type">
       <h3>{fileType.name}</h3>
+      <TooltipIfTruncated customText={fileType.extensions.map(ext => ext).join(', ')}>
```
Comment (ThiefMaster) [inline indico/modules/events/editing/client/js/components/FileManager/index.jsx]: `.map(ext => ext)` doesn't do anything

File: indico/web/client/js/react/components/TooltipIfTruncated.jsx [hunk 2]
```diff
@@ -11,15 +11,21 @@ import PropTypes from 'prop-types';
 export default class TooltipIfTruncated extends React.Component {
   static propTypes = {
     children: PropTypes.any.isRequired,
+    customText: PropTypes.string,
```
Comment (ThiefMaster) [inline indico/web/client/js/react/components/TooltipIfTruncated.jsx]: `customText` vs `text` vs `tooltip`.. any opinions @indico/development-team

File: indico/modules/events/editing/client/js/components/FileManager/index.jsx (hunk 1 above)
Comment (panagiotappl) [reply indico/modules/events/editing/client/js/components/FileManager/index.jsx]: Oops, it was leftover from previous implementations. Thanks for pointing out

File: indico/modules/events/editing/client/js/components/FileManager/FileManager.module.scss [hunk 3]
```diff
@@ -35,16 +35,38 @@
...
+            padding: 0;
+            white-space: nowrap;
+            overflow: hidden;
+            text-overflow: ellipsis;
+
+            li {
+                display: inline;
+            }
+
+            li:not(:last-child)::after {
```
Comment (mic4ael) [inline indico/modules/events/editing/client/js/components/FileManager/FileManager.module.scss]: Maybe move it inside the `li` selector?

```scss

li {
    &:not(:last-child)::after {
```

File: indico/web/client/js/react/components/TooltipIfTruncated.jsx (hunk 2 above)
Comment (mic4ael) [reply indico/web/client/js/react/components/TooltipIfTruncated.jsx]: Hm.. `tooltip` seems like a good name for that (I would be more in favor of `customText` if `element.innerText` had precedence over the prop value, but apparently it is not the case)

Comment (panagiotappl) [reply indico/web/client/js/react/components/TooltipIfTruncated.jsx]: I think I'll just go with `tooltip`. What do you think?

**Pull request merged**